cypress-tests/cypress/e2e/PaymentMethodListUtils/Stripe.js

```diff
@@ -0,0 +1,754 @@
+import { getCustomExchange } from "./Common";
+
+const successfulTestCard = "4242424242424242";
+const successful3DSCard = "4000002500003155";
```

Comment:
You do not need to declare these `const` as the values have been used directly in the below `const`.